fermenter-mda-test-utils/src/main/java/org/technologybrewery/fermenter/mda/test/utils/TestUtil.java

```diff
@@ -0,0 +1,82 @@
+package org.technologybrewery.fermenter.mda.test.utils;
+
+import org.technologybrewery.fermenter.mda.GenerateSourcesHelper;
+import org.technologybrewery.fermenter.mda.element.ExpandedProfile;
+import org.technologybrewery.fermenter.mda.element.Target;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestUtil {
```

Comment:
A: I think we should make this name Fermenter specific to decrease the likelihood of a name collision.  Otherwise adopters will be forced to include the fully qualified name which is a little cumbersome.

Reply:
True, Consumer would definitely have hard time finding corrrect test util. updated name for this class.